utils/find-changed-targets.js

```diff
@@ -43,6 +42,10 @@ const scan = async (root, lines) => {
   ];
 };
 
+function quoteFilePaths(paths) {
+  return paths.map(path => `'${path}'`);
```

Comment:
should probably escape any apostrophes in the path variable to defend against bogus bug bounties

Reply:
Unfortunately, Bazel query does not interpret escaped quotes, causing a syntax error. I've updated the query to use double quotes for consistency, as [recommended](https://bazel.build/query/language#:~:text=However%2C%20be%20careful%20when%20using%20single%20or%20double%20quotes%20in%20target%20names) by bazel:
> However, be careful when using single or double quotes in target names. When quoting one or more target names, use only one type of quotes (either all single or all double quotes).

I've added constraint to disallow use of double quotes in file paths, which we may relax in the future should someone ask to add support for it. I figured it's not worth introducing complexity for now, as this would require running multiple queries using different styles of quotes to delimit words / labels in the bazel query: https://github.com/uber-web/jazelle/pull/158/commits/3d07177d836721ce78b44c1844f2efe8b821b252